src/gebt_poc/element.cpp

```diff
@@ -0,0 +1,43 @@
+#include "src/gebt_poc/element.h"
+
+#include "src/utilities/log.h"
+
+namespace openturbine::gebt_poc {
+
+double CalculateJacobian(
+    const std::vector<Point>& nodes, const std::vector<double>& shape_derivatives
+) {
+    auto dot_product = [](const std::vector<double>& vector_1, const std::vector<double>& vector_2) {
+        auto result = 0.;
+        for (size_t i = 0; i < vector_1.size(); ++i) {
+            result += vector_1[i] * vector_2[i];
+        }
+        return result;
+    };
+
+    auto two_norm = [](const std::array<double, 3>& vector) {
+        return std::sqrt(std::pow(vector[0], 2) + std::pow(vector[1], 2) + std::pow(vector[2], 2));
```

Comment:
didn't you write a math vector recently? why not use that here for these dot and norm calculations?

Reply:
Thanks for catching this - the `Vector` class is a data structure for 3 component physical vectors only, so we could use this for the norm calculation of the Jacobian, but not the dot product calculation since the `std::vector`s consist of varying # of components. Let me know if above doesn't make sense to you.